frontend/src/app/testResults/viewResults/DownloadResultsCsvModal.tsx

```diff
@@ -1,20 +1,19 @@
-import { useEffect, useRef, useState } from "react";
+import { useState } from "react";
 import Modal from "react-modal";
-import { faDownload } from "@fortawesome/free-solid-svg-icons";
+import {
+  faDownload,
+  faCheck,
+  faSpinner,
+} from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { CSVLink } from "react-csv";
-import { ApolloError } from "@apollo/client";
 
 import { showError } from "../../utils/srToast";
 import Button from "../../commonComponents/Button/Button";
-import {
-  GetResultsForDownloadQuery,
```

Comment:
can we clean up [this query](https://github.com/CDCgov/prime-simplereport/blob/965ba9a3a6659fbdf163b23d987ebc6b697cf237/frontend/src/app/testResults/operations.graphql#L167-L167) since we're not using it anymore?